content-src/components/DiscoveryStreamComponents/HorizontalRule/_HorizontalRule.scss

```diff
@@ -0,0 +1,5 @@
+.ds-hr {
+  border: 0;
+  height: 0;
+  border-top: 2px solid var(--newtab-border-secondary-color);
```

Comment:
wolasi confirmed 1px for all horizontal rules, so we can just reuse `border-top: $border-secondary;`